rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc

```diff
@@ -1,181 +1,239 @@
-be used as a relative anchor point for mouse clicks. 'Image' in this
-sense does not only mean 'picture', but also includes the visual
+be used as a relative anchor point for mouse clicks. Image in this
+case does not only mean picture, but also includes the visual
 representation of lettering or control elements like buttons.
 
-The main use cases for the _Image search_ are:
+The most common use cases for _Image search_ are:
 
-* To compute the loading time of a webpage or an application loading
+* Calculating loading times
++
+To compute the loading time of a webpage or an application loading
 data from a distant server, it is not sufficient to wait for the window
 to appear. In such cases, the webpage or application can only be said to
 be loaded when the content is also visualized; that is, when an image
 can be seen.
-* When the measured application is running on a distant system and the
+* Checking for feedback from remote applications
++
+When the measured application is running on a distant system and the
 Workflow has to use a Citrix client, VCN, or a similar remote connection
 tool that only displays the user interface graphically but does not give
 back handles, Image search is the only way to ascertain that the desired
 content has appeared onscreen.
-* Finally, the position of the image can be used to place a mouse click
+* Positioning the mouse relative to an image
++
+Finally, the position of the image can be used to place a mouse click
 relatively to the image, for example to enter a URL in the address bar
 of a web browser, or to enter login information into user and password
 fields.
 
-What the _Image search_ actually does is compare the recorded _Pattern
-image_ to the whole _Search area_. During execution, it takes a
+What the _Image search_ actually does is compare the recorded _Pattern image_ to the whole _Search area_. During execution, it takes a
 screenshot of the _Search area_, applies the selected _Image search_
 algorithm to it, and then compares it to the _Pattern image_. The
-_Search area_ will automatically be adjusted if the executing computer
+_Search area_ adjusts automatically if the executing computer
 has a different screen resolution to the computer the _Workflow_ was
-originally created on. Also, it will use the lowest common color depth
+originally created on. Also, uses the lowest common color depth
 setting, so for example, if you created an _Image search_ with a color
 depth of 32 bits, but the executing computer only has a color depth of
-16 bits, the original _Pattern image_ will be converted on the fly to
+16 bits, the original _Pattern image_ is converted on the fly to
 match the current system's color depth. You can adjust this within the
-*Workflow Properties*.
-//link:#WorkflowProperties[_Workflow Properties_].
+Workflow Properties.
 
 == Properties
 
-* *Compare Precision* This is an additional setting for the search
+* *Compare Precision*

```

Comment:
This "Compare Precision" setting appears in capital letters and no non-capital letters. Is this a label, or should it be in non-capital letters?